auto_submit/lib/requests/github_webhook.dart

```diff
@@ -43,9 +43,17 @@ class GithubWebhook extends RequestHandler {
     print('pullRequest: ${pullRequest.id}');
 
     if (hasAutosubmit) {
-      log.info('Found pull request with auto submit label');
-      // TODO(kristinbi): Publish the pr with 'autosbumit' label to pubsub.
-      await pubsub.publish('auto-submit-queue', pullRequest);
+      log.info('Found pull request with auto submit label.');
+      PullRequest storedPullRequest = PullRequest(
```

Comment:
Here after I get the pull request with the 'autosubmit' label, I will create a new pull request which only stores the useful data we need from the original pull request. As a result, we only need to push this truncated new pull request to the pub sub to save the pubsub's capacity and make it quicker to process message data.

Reply:
I would skip this as it's a micro-optimization. Pub-sub isn't charging us for the message size, and adds additional maintenance burden if `package:github` changes

Reply:
Sure Casey, I will remove this logic and just publish the original pull request.